### PR TITLE
Add Text-Utilities in component library

### PR DESCRIPTION
**Pull request by nikhilmalviya-dev**
Add different types of Text in component library

### CODE REVIEW

File: Text-Utilities/text.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    border-radius: 1rem;
+    border: 1px solid var(--special-color);
+    background-color:rgb(236, 189, 189);
+}
+.side-heading{
+    font-size:1.5rem;
+    font-weight: bold;
+    margin-bottom:-0.5rem;
+}
+#special-btn{
```
Comment (mnath30) [inline Text-Utilities/text.css:53]: Instead of using using ID you can use class names for styling

Comment (nikhilmalviya-dev) [reply Text-Utilities/text.css:53]: ok will do that

**Review by mnath30 — APPROVED** [review]
Code looks good. You can make use of comments in your code for better understanding.

**Review by rushikesh3299 — COMMENTED** [review]
Looks good to Me. You can merge

**Pull request merged**